wpeverest/everest-forms
# Tweak  - Add mark as spam option in entries list.

### All Submissions:






### Changes proposed in this Pull Request:


Previously, there was no direct option to add the entries as Spam without integrating third party plugin such as Akismet Anti-spam. This PR adds the ability to mark an entry as spam.

Closes # .

### How to test the changes in this Pull Request:

1. Create a form from Scratch or use the existing form
2. Perform multiple entries or single as much as you want
3. Now go to **Everest Forms > Entries**
4. Click on the **Mark as Spam** option to mark that specified entry as Spam.
5. To perform the bulk action for marking the entries as spam you must check the entries as much as you want and Click on **Mark as Spam** option under bulk action.
6. Now to Unspam the entry go to **Spams** tab and click on **Remove From Spam** button for single entry also check the Bulk action for Unspamming the entry.

### Types of changes:




### Other information:




### Changelog entry

> Tweak  - Add mark as spam option in entries list.
